packages/usa-table/src/usa-table.twig

```diff
@@ -43,13 +43,15 @@
       {% endfor %}
     {% endfor %}
   </tbody>
+  {% if table_footer %}
```

Comment:
Wrapped the `tfoot` in a conditional statement so that it can be controlled by the `table_footer` Storybook control